ci-operator/step-registry/ipi/conf/gcp/oidc-creds-deprovision/ipi-conf-gcp-oidc-creds-deprovision-commands.sh

```diff
@@ -32,5 +32,6 @@ oc adm release extract --credentials-requests --cloud=gcp --to="/tmp/credrequest
 echo "> Output gcp credentials requests to directory: /tmp/credrequests"
 ls "/tmp/credrequests"
 
+ccoctl_ouptut="/tmp/ccoctl_output"
 echo "> Delete credentials infrastructure created by oidc-creds-provision-provision configure step"
-ccoctl gcp delete --name="${infra_name}" --project="${PROJECT}" --credentials-requests-dir="/tmp/credrequests"
+ccoctl gcp delete --name="${infra_name}" --project="${PROJECT}" --credentials-requests-dir="/tmp/credrequests" 2>&1 | tee "${ccoctl_ouptut}"
```

Comment:
We do not need this change for deprovision step, because the output was not redirected.